cmd/nodecmd/wiz.go

```diff
@@ -247,9 +249,32 @@ func wiz(cmd *cobra.Command, args []string) error {
 	} else {
 		ux.Logger.PrintToUser(logging.Green.Wrap("Devnet %s is successfully created and is now validating subnet %s!"), clusterName, subnetName)
 	}
+
+	if err = deployClusterYAMLFile(clusterName, subnetName); err != nil {
+		return err
+	}
+	ux.Logger.PrintToUser("Successfully deployed cluster information YAML file to external host!")
```

Comment:
imho better wording would be to replace 2 lines
ux.Logger.GreenCheckMarkToUser("Cluster info YAML file  /home/ubuntu/clusterInfo.yaml available for loadtest script")